### ISSUE TITLE
Updater in quilt downloads fabric version

### ISSUE THREAD

**Issue by MenacingPerson** [Wynntils/Wynntils#1354]
Sometimes the fabric version is broken on quilt, so this is concerning.

It can probably be fixed by changing https://github.com/Wynntils/Artemis/blob/main/.github/workflows/pre-release.yml to have `quilt, fabric, forge` instead of `fabric, forge, quilt`, and I can PR that if you want.

**Comment by MenacingPerson** [comment]
(Btw, the same goes for the PrismLauncher mod updater, and probably has the same fix.)

**Comment by magicus** [comment]
This sounds concerning, but I am not sure I fully understand. Perhaps a PR will help show better what you mean, so please open one.

**Comment by kristofbolyai** [comment]
No, that order does not matter. This is to be fixed in Athena.

**Comment by MenacingPerson** [comment]
What about the prism launcher updater? There is no reason it would prefer fabric > quilt right?

**Comment by kristofbolyai** [comment]
> What about the prism launcher updater? There is no reason it would prefer fabric > quilt right?
> 
> 

Is this issue about the mod's updater or prism launcher updater to begin with?

**Comment by MenacingPerson** [comment]
Both, kind of. They face similar issues, both download the wrong mod version. It might be either prism launcher's or wynntils's fault. I don't know.

**Comment by kristofbolyai** [comment]
Firstly, we are not here to fix prism launcher, if they can't tell the difference between quilt and fabric from semver, their problem. 

As for our updater, the update file link is served from athena, from user agents. So this is a problem there. 

And for the fix of changing orders, I can't see any solid reason as to why it would fix it, or any guarantee about not breaking fabric, which is arguably more important.

**Comment by magicus** [comment]
I'm looking at Modrinth. This is claimed to be the latest Quilt version: https://modrinth.com/mod/wynntils/version/v0.0.2-alpha.359. Have you configured Prism Launcher to fetch metadata from Modrinth or Curseforge?

**Comment by MenacingPerson** [comment]
> And for the fix of changing orders, I can't see any solid reason as to why it would fix it, or any guarantee about not breaking fabric, which is arguably more important.

Ok I apologize for guessing solutions, I was severely wrong here.

**Comment by MenacingPerson** [comment]
> I'm looking at Modrinth. This is claimed to be the latest Quilt version: https://modrinth.com/mod/wynntils/version/v0.0.2-alpha.359. Have you configured Prism Launcher to fetch metadata from Modrinth or Curseforge?

Modrinth.

**Comment by kristofbolyai** [comment]
We will look into fixing this server side.

**Comment by magicus** [comment]
Ok, then I don't know why Prism Launcher is not downloading that file. I also double-checked that it is indeed the Quilt version of the mod.

**Comment by MenacingPerson** [comment]
I will report this to prism launcher.

Edit: This has already been reported using wynntils as an example https://github.com/PrismLauncher/PrismLauncher/issues/818
Sorry for wasting some of your time...

**Comment by kristofbolyai** [comment]
Looks like this is not an issue for us.

**Comment by MenacingPerson** [comment]
So has the in-game wynntils updater been fixed in quilt?

**Comment by kristofbolyai** [comment]
Didn't we agree this is a PrismLauncher issue?

**Comment by MenacingPerson** [comment]
I reported both the prism thing and the ingame thing together. So apologies for the confusion, I will resolve it here:

The in-game updater, built into artemis, unrelated to prism launcher, on quilt loader using the proper wynntils jar for quilt, when used, downloads and installs the wynntils jar file for fabric.
This may be an issue because I've had some prior versions of artemis crashing on quilt if I use the fabric jar.

... or so I reported.

I'll re-test it and report back if this is still an actual issue or if it was some dumb mistake by me.
